packages/dht/src/connection/WebRTC/IWebRtcConnection.ts

```diff
@@ -0,0 +1,22 @@
+export enum Event {
+    LOCAL_DESCRIPTION = 'streamr:dht:webrtcconnection:localdescription',
+    LOCAL_CANDIDATE = 'streamr:dht:webrtcconnection:localcandidate',
+}
+
+export enum RtcDescription {
+    OFFER = 'offer',
+    ANSWER = 'answer'
+}
+
+export interface IWebRtcConnection {
```

Comment:
Let's do it

Reply:
This would unfortunately be impossible with EventEmitter3, as the IConnection events of the interface 'IWebRtcConnection extends IConnection' would be of different type than events of IConnection, and thus it would not be possible to wait for IConnection events on NodeWebRtcConnection class.